docs/configuration/multiple-delivery-clients.md

```diff
@@ -1,36 +1,128 @@
+    {
+        services.AddMultipleDeliveryClientFactory
+        (
+            factoryBuilder => factoryBuilder
+                .AddDeliveryClient
+                (
+                    "projectA",
+                    deliveryOptionBuilder => deliveryOptionBuilder
+                        .WithProjectId("<A_PROJECT_ID>")
+                        .UseProductionApi()
+                        .Build(),
+                    optionalClientSetup =>
+                        optionalClientSetup.WithTypeProvider(new ProjectAProvider())
+                )
+                .AddDeliveryClient(
+                    "projectB",
+                    deliveryOptionBuilder => deliveryOptionBuilder
+                        .WithProjectId("<B_PROJECT_ID>")
+                        .UseProductionApi()
+                        .Build(),
+                    optionalClientSetup =>
+                        optionalClientSetup.WithTypeProvider(new ProjectBProvider())
+                )
+        );
+    }
+}
 ```
 
-## Adding Autofac to the hosting pipeline
-To enable Autofac DI resolution, you need to [add Autofac to your hosting pipeline](https://autofaccn.readthedocs.io/en/latest/integration/aspnetcore.html#asp-net-core-3-0-and-generic-hosting) as follows:
+### Registering Cached client
+
+> To be able to use the [.NET SDK caching layer](../retrieving-data/caching.md), you need to install `Kontent.Ai.Delivery.Caching` package.
+
+If you want to use the cached client, you can use the `AddCachedDeliveryClient` method instead of `AddDeliveryClient`
 
 ```csharp
-public class Program
+public class Startup
 {
-     public static IHostBuilder CreateHostBuilder(string[] args) =>
-              Host.CreateDefaultBuilder(args)
-                  .UseServiceProviderFactory(new AutofacServiceProviderFactory()) // Add this line
-                  .ConfigureWebHostDefaults(webBuilder =>
-                  {
-                      webBuilder.UseStartup<Startup>();
-                  });
+    // ...
+
+    public void ConfigureServices(IServiceCollection services)
+    {
+        services.AddMultipleDeliveryClientFactory
+        (
+                factoryBuilder => factoryBuilder
+                    .AddDeliveryClientCache
+                    (
+                        "projectA"
+                        deliveryOptionBuilder => deliveryOptionBuilder
+                            .WithProjectId(ClientAProjectId)
+                            .UseProductionApi()
+                            .Build(),
+                        CacheManagerFactory.Create(
+                            new MemoryCache(new MemoryCacheOptions()),
+                            Options.Create(new DeliveryCacheOptions
+                            {
+                                CacheType = CacheTypeEnum.Memory
+                            })),
+                        optionalClientSetup =>
+                            optionalClientSetup.WithTypeProvider(new ProjectAProvider())
+                    )
```

Comment:
It seems that
`.Build()`
is missed here

Reply:
ae78d43